app/src/main/res/values-zh-rCN/strings.xml

```diff
@@ -507,12 +507,30 @@
     <string name="no_one_watching">无人在线观看</string>
     <plurals name="watching">
         <item quantity="one">%s 人在观看</item>
-        <item quantity="other"></item>
+        <item quantity="other"/>
```

Comment:
Why are you deleting and not translating ?

Reply:
I don't know either, the file was automatically written by Weblate.

Reply:
I think it's a bug then. Look at [the english string on weblate](https://hosted.weblate.org/translate/newpipe/strings/en/?q=watching&offset=2), it shows singular and plurial, and haven't been changed for a month. You should fix with "%s plural_for_watching_in_chinese".

Reply:
I am busy, but I will fix it soon.

Reply:
Besides, Chinese language has no singular and plurial grammar.

Reply:
Now，they were solved.